installer/datafiles/base_omsagent.data

```diff
@@ -23,6 +23,7 @@ MAINTAINER:              'Microsoft Corporation'
 /etc/opt/microsoft/omsagent/sysconf/installinfo.txt;                    installer/conf/installinfo.txt;                        644; root; root; conffile
 /etc/opt/microsoft/omsagent/sysconf/omi_mapping.json;                   installer/conf/omi_mapping.json;                       644; root; root
 /etc/opt/microsoft/omsagent/sysconf/omsagent.d/change_tracking.conf;    installer/conf/omsagent.d/change_tracking.conf;        644; root; root
+/etc/opt/microsoft/omsagent/sysconf/omsagent.d/collectd.conf;           installer/conf/omsagent.d/collectd.conf;               644; root; root
```

Comment:
Does this need to be copied to the standard conf directory (owned by/editable/usable by OMS itself)? See later in this file for how we handle omsagent.conf.

The sysconf directory is normally a "staging area" by the installer, but we normally copy to the conf directory during install so the file is actually usable.